imap_processing/lo/l1b/lo_l1b.py

```diff
@@ -5,6 +5,7 @@
 from pathlib import Path
 
 import numpy as np
+import pandas
```

Comment:
Standard is to import it as this. I realize it is only used in one place but would be nice to keep it similar throughout the codebase.
```suggestion
import pandas as pd
```